.buildkite/shared-pipeline-vars

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+# This file is `source`'d before calling `buildkite-agent pipeline upload`, and can be used
+# to set up some variables that will be interpolated in the `.yml` pipeline before uploading it.
+
+CI_TOOLKIT_PLUGIN_VERSION="2.18.2"
+RUBY_VERSION=$(cat .ruby-version)
```

Comment:
I think it'd be good to use the same version in CI as the project uses locally.

But I'm also open to the idea of decoupling them in case we run into some situation where we want to use a version locally for which Docker has no image.

However, if we run into that scenario, maybe we should take a step back and ask why? Given's Ruby adoption, a version with no Docker image seems odd. Maybe if we end up in that scenario we are doing something odd ourselves?